### ISSUE TITLE
[FEATURE REQUEST] Custom `thumb` support when sending files.

### ISSUE THREAD

**Issue by DaviSMoura** [yagop/node-telegram-bot-api#1035]
### Introduction

This package don't support `thumb` param when sending any file (its supported for TG Bot API).
Yes, i know Telegram API generate itself thumb for all videos but it can fail sometimes and Telegram API accepts **custom** thumb so why shouldn't a package that proposes to run the official telegram api support it too? haha. That's why im requesting this feature.

### Example
```javascript
await bot.sendVideo("1251429262", "./video.mp4", {
  thumb: './video_custom_thumb.jpg' // <--- this is not supported by this package but its supported by telegram official api
})
// ----/\ this example is when sending video but can works with almost all type of files, including sendMediaGroup
```

**Comment by danielperez9430** [comment]
You can do it, but is:

`{ thumb: "attach://./video_custom_thumb.jpg" }`

<img width="857" alt="image" src="https://user-images.githubusercontent.com/10388515/208638141-93600172-8f0b-4609-961d-724f39466818.png">

**Comment by danielperez9430** [comment]
Fixed on `v0.61.0`
